samples/springboot3/graphql-pet-store/template.yml

```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Example Pet Store API written with SpringBoot3, Spring for GraphQl and the aws-serverless-java-container library
+
+Globals:
+  Api:
+    # API Gateway regional endpoints
+    EndpointConfiguration: REGIONAL
+
+Resources:
+  PetStoreFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: com.amazonaws.serverless.sample.springboot3.StreamLambdaHandler::handleRequest
+      Runtime: java21
+      CodeUri: .
+      MemorySize: 1512
```

Comment:
I saw the other springboot pet-store example already has `1512`, but it's a weird number. Normally the memory comes in like powers of 2 (`512` is in some other examples). I feel like someone just added a `1` when they wanted more memory to end up with the `1512`. But it looks better to use `1024` if that's enough memory (that's what the `alt-petstore` uses).

Reply:
Updated.